### PR TITLE
Update PPP - Wei Xin

### PR DESCRIPTION
**Pull request by Wei-Xinn**
PR update for my PPP.

### CODE REVIEW

File: docs/team/wei-xinn.md [hunk 1]
```diff
@@ -5,30 +5,36 @@ title: Wei Xin's Project Portfolio Page
...
   * Justification: ContaX's extensive feature set may set a learning curve for new users. The Onboarding guide serves as starting point for new users by introducing the basic functions in a manner that is simple to understand.
 
-* **Code contributed**:
-  * Functionality, documentation and tests for Onboarding Guide
-
-* **Project management**:
-  * `to be added soon`
+* **New Feature**: Added the ability to find appointments by name
+  * What it does: Allows users to find quickly find appointments using either the appointment name or person name
+  * Justification: This feature expands the ways in which a user can locate a specific appointment, allowing for more efficient management.
```
Comment (sharpstorm) [inline docs/team/wei-xinn.md:20]: Consider the following phrasing.
```suggestion
  * Justification: This feature expands on the ways in which a user can locate specific appointments, allowing for more efficient access and subsequent management of Appointments.
```

Comment (Wei-Xinn) [reply docs/team/wei-xinn.md:20]: Thanks for the suggestion. Updated

File: docs/team/wei-xinn.md [hunk 2]
```diff
@@ -5,30 +5,36 @@ title: Wei Xin's Project Portfolio Page
...
+  * Worked on the Onboarding Guide and Help Window component
+    * Includes components in `UI` and `Model`
 
+* **Project management**:
+  * Code reviews
+  
 * **Documentation**:
   * User Guide:
-    * Added documentation for onboarding guide
+    * Added documentation for Onbarding Guide, Help Window and Find Appointment command
```
[the remark this answers is not in the document]
Comment (Wei-Xinn) [reply docs/team/wei-xinn.md:35]: Fixed

**Review by sharpstorm — APPROVED** [review]
LGTM.

File: docs/team/wei-xinn.md [hunk 3]
```diff
@@ -5,33 +5,39 @@ title: Wei Xin's Project Portfolio Page
...
+* **New Feature**: Added the ability to find appointments by name
+  * What it does: Allows users to find quickly find appointments using either the appointment name or person name
+  * Justification: This feature expands on the ways in which a user can locate specific appointments, allowing for more efficient access and subsequent management of Appointments.
 
 * **Enhancements to existing features**:
-  * `to be added soon`
+  * Revamped Help Window to display meaningful information on all commands in ContaX
 
+* **Code contributed**:
+  * 3000+ LoC contributed to the project
```
Comment (sharpstorm) [inline docs/team/wei-xinn.md:26]: Add a link to reposense, it is a requirement.

Comment (Wei-Xinn) [reply docs/team/wei-xinn.md]: Added thanks

**Review by dandaandaaaaaan — APPROVED** [review]
lgtm

**Pull request merged**